src/brevitas/graph/equalize.py

```diff
@@ -93,22 +108,65 @@ def _select_scale_computation_fn(
         raise RuntimeError(f"Scale computation type {scale_computation_type} not supported")
 
 
-def _channel_range(inp: torch.Tensor) -> torch.Tensor:
-    mins, _ = inp.min(dim=1)
-    maxs, _ = inp.max(dim=1)
+class activation_equalization_mode:
+
+    def __init__(self, graph_model, alpha, add_mul_node, layerwise=False, enabled=True) -> None:
+        self.graph_model = graph_model
+        self.alpha = alpha
+        self.enabled = enabled
+        self.add_mul_node = add_mul_node
+        if layerwise:
+            self.add_mul_node = True
+            self.graph_act_eq = LayerwiseActivationEqualization(self.graph_model)
+        else:
+            self.graph_act_eq = GraphActivationEqualization(self.graph_model, self.add_mul_node)
```

Comment:
We need to verify that it's indeed a GraphModule. Needs to be done both against the backport and the pytorch one.